CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 
 # This is the boilerplate for Mbed OS
 
-cmake_minimum_required(VERSION 3.18.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
```

Comment:
Travis will fail if it fetches from pip (this version not yet there unfortunately)

I can do:

```
        # Fetch the 3.19.0
        - DEPS_DIR="${TRAVIS_BUILD_DIR}/deps"
        - mkdir ${DEPS_DIR} && cd ${DEPS_DIR}
        - travis_retry wget --no-check-certificate https://cmake.org/files/v3.19/cmake-3.19.0-rc2-Linux-x86_64.tar.gz
        - tar -xvf cmake-3.19.0-rc2-Linux-x86_64.tar.gz > /dev/null
        - mv cmake-3.19.0-rc2-Linux-x86_64 cmake-install
        - PATH=${DEPS_DIR}/cmake-install:${DEPS_DIR}/cmake-install/bin:$PATH
        - cd ${TRAVIS_BUILD_DIR}
```
However this would be just partial fix for now. I'll talk to test team to include 3.19.0 in the jenkins and this can wait for CI update to land